forms-mit/standards.html

```diff
@@ -1,167 +1,213 @@

-        </label>

-        <input type="hidden" name="r_lastname" value="-Please enter your last name.">

-      </p>

-      <p>

-        <label for="firstname">Your first name<br>

-          <input name="firstname" type="text" id="firstname" size="40" onChange="saveValue(this)">

-        </label>

-      </p>

-      <p>

-        <label for="phone">Your phone number<br>

-          <input name="phone" type="text" id="phone" size="40" onChange="saveValue(this)">

-        </label>

-      </p>

-      <p>

-        <label for="mitstatus">Your status <abbr class="required" title="required">*</abbr><br>

-          <select name="mitstatus" id="mitstatus" onChange="saveValue(this)" aria-required="true">

-            <option value="">Please indicate your status</option>

-            <option value="UG">MIT Undergrad Student</option>

-            <option value="GRAD">MIT Grad Student</option>

-            <option value="FAC">MIT Faculty</option>

-            <option value="RS">MIT Research Staff</option>

-            <option value="STAFF">MIT Staff</option>

-            <option value="OTHER">MIT Other</option>

-          </select>

-        </label>

-        <input type="hidden" name="r_mitstatus" value="-Please indicate your status">

-      </p>

-      <p>

-        <label for="department">Your department <abbr class="required" title="required">*</abbr><br>

-          <select name="department" id="department" onChange="saveValue(this)" aria-required="true">

-            <option value="" selected>- choose department -</option>

-            <!--#include virtual="../forms-text/departments.txt" -->

-          </select>

-        </label>

-        <input type="hidden" name="r_department" value="-Please indicate your department">

-      </p>

-      <p>

-        <label for="lab">Lab or Center name <br>

-          <input name="lab" type="text" id="lab">

-        </label>

-      </p>

-    </fieldset>

-    <p id="submit">

-      <input type="submit" value="Request a Standard">

-    </p>

-  </form>

-

-  <!--#include virtual="/includes/footer-semantic.html" -->

-</div> <!-- /end container -->

-</body>

-</html>

-

+<!DOCTYPE html>
+<!--[if lte IE 9]><html class="no-js lte-ie9" lang="en"><![endif]-->
+<!--[if !(IE 8) | !(IE 9) ]><!-->
+<html lang="en-US" class="no-js">
+<!--<![endif]-->
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Standards Request Form: MIT Libraries</title>
+
+  <!--#include virtual="/mitlib-style/inc/assets-site-head.html" -->
+ <!--#include virtual="../includes/base_scripts.html" -->
+  <!-- form specific stuff -->
+  <style type="text/css">
+  .hidden {
+    display: none;
+  }
+  .box {
+    margin-bottom: 20px;
+    border: 3px solid rgba(51, 139, 197, 0.3);
+    background-color: #fff;
+    padding: 20px 30px;
+  }
+
+  .box blockquote {
```

Comment:
Did not remove the box styles for standards.html because the next comment was to use the styles in formatting the page